### PR TITLE
Abandoned tasks now show abandonment date

### PR DESCRIPTION
**Pull request by AbdusShaikh**

### CODE REVIEW

**Comment by AbdusShaikh** [comment]
To test:
1. Create a task
2. Abandon this task ensuring the abandonment date is different from the due date
3. Confirm the abandonment is appearing in place of the due date in the Tasks page.

File: packages/webapp/src/containers/Task/TaskCard/index.jsx [hunk 1]
```diff
@@ -33,6 +33,13 @@ const TaskCard = ({
   const user = useSelector(userFarmSelector);
   const immutableStatus = ['completed', 'abandoned'];
 
+  let trueDate = completeOrDueDate;
+  console.log(props);
```
Comment (seinnlee) [inline packages/webapp/src/containers/Task/TaskCard/index.jsx:37]: I think you forgot to remove the console.log

**Review by seinnlee — CHANGES_REQUESTED** [review]
If `trueDate` is only going to be passed into one component, I think it's better to write this mapping in `TaskCard.jsx` component rather than in `index.jsx` file.

**Comment by AbdusShaikh** [comment]
> If `trueDate` is only going to be passed into one component, I think it's better to write this mapping in `TaskCard.jsx` component rather than in `index.jsx` file.

I have made a commit making these changes

**Comment by smattingly** [comment]
@AbdusShaikh Since @seinnlee is out today, I will pick up the review of this from here.
Also, merge conflicts have developed for this branch. If you know how to address that, please do. If not, reach out and we can go through that.

**Review by smattingly — APPROVED** [review]
:+1:

**Pull request merged**